src/parser/jobs/mnk/modules/Brotherhood.tsx

```diff
@@ -0,0 +1,89 @@
+import {t, Trans} from '@lingui/macro'
+import {ActionLink} from 'components/ui/DbLink'
+import {StatusRoot} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+const FULL_PARTY_SIZE = 8
+
+export class Brotherhood extends Analyser {
+	static override debug = false
```

Comment:
I'm fine with trailing debugs as long as it's set to false, but it doesn't look like there's any debug statements in this file anymore?